apps/omg_child_chain/lib/omg_child_chain/supervisor.ex

```diff
@@ -33,13 +36,19 @@ defmodule OMG.ChildChain.Supervisor do
   end
 
   def init(:ok) do
-    # prevent booting if contracts are not ready
-    :ok = RootChain.contract_ready()
+    :ok = Client.node_ready()
```

Comment:
we couldn't have started if contracts were not ready because of `SetContract` release task